### ISSUE TITLE
Refuses to stay open during start up

### ISSUE THREAD

**Issue by KirkKitLHTC** [Sheights/StarboundSimpleVoreMod#192]
I know some people have this problem too, but I don't know what version of Starbound the mod's recent update is compatible with. Basically, the game closes it's self as the fish logo slowly vanishes. I tried with the 9.5 version of the mod on 1.4 of Starbound, AND the master copy. I even tried using the Nexus mod loader and the problem continued. Someone just please solve this or update the mod or something.

Also the mod does not have a .modinfo file.
[starbound.log](https://github.com/Sheights/StarboundSimpleVoreMod/files/4834508/starbound.log)

**Comment by florancetheavian** [comment]
this I have absolutely no background in. nor have I had it happen to me. I'll look into the problem and research. it might take a while due to starbound not being as popular anymore, but I'll try my hardest to help.

**Comment by florancetheavian** [comment]
ok, you said there's no crash log, and Starbound refuses to stay open. after a lot of thought, I think I k ow the problem. this usually happens to me when there is a duplicate file, usually found in one of the base folders, or the first 1 or 2 sub-folders within those. I hope this helps you!

**Comment by KirkKitLHTC** [comment]
@florancetheavian By one of the base folders do you mean in the main mod folder or the game's folder?

**Comment by florancetheavian** [comment]
the main mod folders

**Comment by KirkKitLHTC** [comment]
@florancetheavian What files would usually be duplicated?

**Comment by florancetheavian** [comment]
it all depends. I'm not sure which files are usually duplicated because I've only come across this problem 3 times. but from my experience, it was a zip folder, and a separate folder with the same folder in both. I hope that makes sense.

**Comment by KirkKitLHTC** [comment]
@florancetheavian It doesn't make that much sense but I get the gist of it sort of.


I didn't find any duplicate zip folders and going in every file and looking for a duplicate is taking a while.
Would the duplicates be in the "StarboundSimpleVoreMod" and "StarboundSimpleVoreModVSO" or one of the SSVMSPOV files?

**Comment by florancetheavian** [comment]
it's usually the "StarboundSimpleVoreMod" or "StarboundSimpleVoreModVSO"

**Comment by CodeSearcher59** [comment]
Are you using FU? This mod can't be installed with it now because both mods add an item called "deliciousmeat", at least this is what I could see in the logs.

**Comment by florancetheavian** [comment]
no, I personally dislike FU. it caused too much lag, and crashed my game a lot. so no, I don't use and still don't use FU.

**Comment by KirkKitLHTC** [comment]
No I'm not using FU @CodeSearcher59

**Comment by WolfSteele** [comment]
okay so here is what i found in your log:
main problem is found in line 240:
[20:31:08.233] [Error] Application: exception thrown, shutting down: (ItemException) Duplicate item name 'yoshichest' found

what kinda confuses me though is that you are using vortex to intall mods as the fastest way is to just unzip/unrar the downloaded mods and put the right folders in the mods folder AND that you actually have the downloaded rar file "StarboundSimpleVoreMod.rar" in the mods folder

I am using more then 900+ mods both Steam and from other sauces (SimpleVoreMod / Sexbound / Other) 
though since i didn't play for around 2 months now it at least shows that it is possible

**Comment by KirkKitLHTC** [comment]
@WolfSteele I finally got it working now. The Yoshi race workshop mod was the problem. I unsubscribed to it and the mod works now. Thanks for the help finding the problem.

**Comment by florancetheavian** [comment]
@WolfSteele nice job! glad you could help him. @KirkKitLHTC have fun on starbound! ^^

**Comment by KirkKitLHTC** [comment]
@florancetheavian Thanks!
